### PR TITLE
Flat Icons

### PR DESCRIPTION
**Pull request by Paszt**
Changed icons to flat style.  Added new "loading" icon to signify that
song info is being loaded from last.fm. Ref #650, #197.

### CODE REVIEW

**Comment by inverse** [comment]
I dig these changes! :+1: looks good!

File: core/background/controller.js [hunk 1]
```diff
@@ -294,6 +297,7 @@ define([
 
 				// re-send song to pipeline
 				if (data.artist || data.track) {
+					pageAction.setSongLoading(currentSong);
```
Comment (inverse) [inline core/background/controller.js]: :+1: nice!

**Comment by david-sabata** [comment]
Awesome, thanks! :+1:

**Comment by inverse** [comment]
Damn, looks good!

**Pull request merged**